frontend/src/utilities/useValidation.ts

```diff
@@ -78,6 +78,14 @@ export const useValidation = <T>(data: T, schema: z.ZodType<T>): ValidationConte
     );
   }, []);
 
+  const getAllValidationIssues = React.useCallback((path?: (string | number)[]) => {
+    const error = resultRef.current?.error;
+    if (!error) {
+      return [];
+    }
+    return error.issues.filter((issue) => (path ? issue.path.join('.') === path.join('.') : true));
+  }, []);
```

Comment:
Don't we need to add `resultRef` in the dependency array?

Reply:
no. its a ref so it doesnt. I followed the use case above anyway